server/server.js

```diff
@@ -103,6 +103,7 @@ function server(serviceManager, options) {
     // take care to not callback until any actions are complete.
     if (!callback) {
       callback = function(err) {
+        // XXX(sam) shouldn't err of an err-first already be an Error() object?
         if (err) throw Error(err);
```

Comment:
Yes, thats a bug but not the one we were seeing in my demo. A throw here would cause process to exit.